config/ci.mocha.node.opts

```diff
@@ -0,0 +1,2 @@
+--reporter dot
+--exit
```

Comment:
can you please explain the 2 options? Is it possible to put comments in the file?

Reply:
I think we can, although that requires us to upgrade mocha to the latest version (5.2.0) in all of our packages. We have 5.0.5 currently.

I can create another pull request to bump the mocha version in all of our SDK packages. This pull request will probably need approvals from all the SDK owners. Let me know if you want me to proceed with it.